src/components/pets/CreateImageUpload.js

```diff
@@ -7,9 +7,14 @@ import pawSrc from '../../images/taskpaw.png'
 CreateImageUpload.propTypes = {
   previewImage: PropTypes.object.isRequired,
   handleImageUpload: PropTypes.func.isRequired,
+  loading: PropTypes.number.isRequired,
```

Comment:
for loading I would expect a bool, but as it looks like you can have >0 the way you use it is a number!?

Reply:
Firebase Storage actually sends a number to let you know the % loaded :)